com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -540,6 +541,7 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
 
                         using (new ProfilingScope(null, Profiling.Pipeline.beginCameraRendering))
                         {
+                            UpdateCameraStereoMatrices(baseCameraData);
```

Comment:
I think we should call UpdateCameraStereoMatrices after InitializeAdditionalCameraData. And for overlay camera, shouldn't we use overlay camera data here?

Reply:
Good point, done!